src/comet/driver/photonic/f3000.py

```diff
@@ -0,0 +1,68 @@
+from comet.driver.generic.light_source import LightSource
+
+__all__ = ["F3000"]
+
+
+class F3000(LightSource):
+    """Class for controlling Photonics F3000 LED light sources"""
+
+    @property
+    def brightness(self) -> int:
+        """Returns current brightness of light source
+            (0-100 percent)
+
+        Returns:
+            int . Brightness of light source in percent
+        """
+
+        self.resource.write("B?")
+        response = self.resource.read()
+
+        return int(response.replace("B", ""))
+
+    @brightness.setter
+    def brightness(self, brightness: int):
+        """Set brightness of light source in percent
+
+        Args:
+            brightness (int): Brightness to set in percent
+        """
+        brightness = max(0, min(100, brightness))
+
+        self.resource.write(f"B{brightness}")
+        self.resource.read()
+
+    @property
+    def light_enabled(self) -> bool:
+        """Get current state of shutter (light source)
+
+        Returns:
+            str: State of light source (1 on, 0 off)
+        """
+
+        self.resource.write("S?")
+        response = self.resource.read().replace("S", "")
+
+        if int(response) == 1:
+            return 0
+        elif int(response) == 0:
+            return 1
```

Comment:
mypy type check complains:
- return type is not `bool`
- `None` is returned if neither is true

Possible solution:
```python
return int(response) == 0  # 0: on, 1: off
```

Reply:
Fixed in [78a8414](https://github.com/hephy-dd/comet/pull/75/commits/78a8414ec4fbac0816089eae642584fd7f619272)